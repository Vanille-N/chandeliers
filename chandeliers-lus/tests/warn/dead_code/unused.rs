#![deny(dead_code)]

chandeliers_lus::decl! {
    node count1() returns (n : int);
    let n = 0 fby n + 1; tel;

    node count2() returns (n : int);
    let n = 0 fby n + 1; tel;

    node count3() returns (n : int);
    let n = 0 fby n + 1; tel;

    const X: int = 0;

    #[main]
    node system() returns ();
    let
        assert count1() = count3();
    tel
}
