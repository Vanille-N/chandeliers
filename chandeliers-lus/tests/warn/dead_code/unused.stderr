error: method `step` is never used
 --> tests/warn/dead_code/unused.rs:7:10
  |
7 |     node count2() returns (n : int);
  |          ^^^^^^ method in this implementation
  |
note: the lint level is defined here
 --> tests/warn/dead_code/unused.rs:1:9
  |
1 | #![deny(dead_code)]
  |         ^^^^^^^^^
